File: Variables Operaciones/1_09.AreaCirculoCs/AreaCirculo.cs
using System;
/*
1.9. Crea un proyecto llamado “ÁreaCirculo” donde se ingresa el radio del círculo imprime el
área. Area = pi*r^2
 */

namespace _1_09.AreaCirculoCs
{
    class AreaCirculo
    {
        static void Main(string[] args)
        {
            Single radio = radioCirculo();
            Console.WriteLine("El Área es : {0}", Math.PI * (Math.Pow(radio, 2)));


        Single radioCirculo(){
            Console.Write("Ingrese radio del círculo : ");
            return Convert.ToSingle(Console.ReadLine());
        }
        }
    }
}

File: Variables Operaciones/1_09.AreaCirculoVb/AreaCirculo.vb
Imports System
'1.9. Crea un proyecto llamado “ÁreaCirculo” donde se ingresa el radio del círculo imprime el
'área. Area = pi*(r^2)
Module AreaCirculo
    Sub Main(args As String())
        Dim radio As Single = radioCirculo()
        Console.WriteLine("El Área es : {0}", Math.PI * (radio ^ 2))
    End Sub


    Function radioCirculo() As Single
        Console.Write("Ingrese radio del círculo : ")
        Return Console.ReadLine()
    End Function
End Module
